Plugins/InstallSafeDITool/InstallCLIPluginCommand.swift

```diff
@@ -0,0 +1,110 @@
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Foundation
+import PackagePlugin
+
+@main
+struct InstallSafeDITool: CommandPlugin {
+    func performCommand(
+        context: PackagePlugin.PluginContext,
+        arguments _: [String]
+    ) async throws {
+        guard let safeDIOrigin = context.package.dependencies.first(where: { $0.package.displayName == "SafeDI" })?.package.origin else {
+            Diagnostics.error("No package origin found for SafeDI package")
+            exit(1)
+        }
+        switch safeDIOrigin {
+        case let .repository(url, displayVersion, _):
+            // As of Xcode 16.0 Beta 6, the display version is of the form "Optional(version)".
+            // This regular expression is duplicated by SafeDIGenerateDependencyTree since plugins can not share code.
+            guard let versionMatch = try /Optional\((.*?)\)|^(.*?)$/.firstMatch(in: displayVersion),
+                  let versionSubstring = versionMatch.output.1 ?? versionMatch.output.2
```

Comment:
Why checking `.1` as well as `.2`?

Reply:
Ditto in the copied code in the other plugin

Reply:
I'm falling back to a version with no surrounding `Optional(...)` in case this gets fixed. `.1` gets us the optional match. `.2` gets us the match without the optional.